recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ requirements:
   host:
     - python
     - pip
-    - pytest-runner
+    - pytest-runnerß
```

Comment:
typo?

Reply:
> usually a PR gets triggered when there is a new version on pypi ... but it takes some time :)
> 
> Have the requirements changed?

I know, I thought I would do it manually as it was taking a while :) 

This happened when I rerendered - I didn't change it so I assume it's fine?

Reply:
Oh but it doesn't like it, I'll take it out